pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
let's add this file to the scripts directory

Reply:
Okay